node/lib/util/commit.js

```diff
@@ -301,23 +362,26 @@ exports.commit = co.wrap(function *(metaRepo, all, metaStatus, message) {
     const subCommitters = Object.keys(submodules).map(commitSubmodule);
```

Comment:
Yeah, I know the library is old -- I just googled it at random.  It just seems *weird* to have to write code like this.  But I guess I'll drop it.